### DISCUSSION TITLE
Localization for Tui Calendar form popup date picker

### DISCUSSION THREAD

**Discussion by thomasdevries** [nhn/tui.calendar#1311] [Q&A] [answered]
## Version
latest

## Test Environment
Google Chrome

## Current Behavior
When use-form=popup=true, the form popup with the tui date picker works fine, however I would like to change the format from MM-DD-YYYY to DD-MM-YYYY, becasause I live in Europe and that's the standard here. Also I would like to change the months and days to in my case Dutch. Is there a way to do that? I am using the the vue-calendar.

**Comment by adhrinae** [comment, marked as answer]
Unfortunately, we don't support any localization for the form popup.
You'll need to create your own popup to do that.

Similar to #860.

**Reply by thomasdevries** [reply]
Thanks for the answer.
